spark/dl/src/test/scala/com/intel/analytics/bigdl/keras/MeanAbsolutePercentageCriterionSpec.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.keras
+
+import com.intel.analytics.bigdl.nn.{Abs, MeanAbsolutePercentageCriterion}
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.utils.serializer.{ModuleLoader, ModulePersister}
+
+import scala.util.Random
+
+class MeanAbsolutePercentageCriterionSpec extends KerasBaseSpec {
+  "MeanAbsolutePercentageCriterion" should "be ok" in {
+    val kerasCode =
+      """
+        |input_tensor = Input(shape=[10])
+        |target_tensor = Input(shape=[10])
+        |loss = mean_absolute_percentage_error(input_tensor, target_tensor)
```

Comment:
it should be 
mean_absolute_percentage_error(target_tensor, input_tensor)

Reply:
no, it will be error if mean_absolute_percentage_error(target_tensor, input_tensor). Also for other keras criterions.